ideasbox/library/migrations/0006_auto_20150521_1207.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
You should squash all those migrations in only one.
You can use [squashmigrations](https://docs.djangoproject.com/en/1.7/ref/django-admin/#django-admin-squashmigrations), but maybe the easiest way as soon as those migrations have not been merged into master is to just remove the files and rerun `makemigrations`, and then force push.